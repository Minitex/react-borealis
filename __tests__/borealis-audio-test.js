import React from 'react';
import renderer from 'react-test-renderer';
import BorealisAudio from '../src/borealis-audio-player';

describe('BorealisAudio Tests', () => {
  it('renders an audio', () => {
    const config = {
      audio: {
        src: 'http://reflections.mndigital.org/utils/getstream/collection/mhs/id/1919',
        thumbnail: 'http://reflections.mndigital.org/utils/getthumbnail/collection/jhs/id/885',
        transcript: {
          texts: ['Audio Transcript Here'],
          label: 'Audio',
        },
      },
    };

    const component = renderer.create(<BorealisAudio config={config} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
