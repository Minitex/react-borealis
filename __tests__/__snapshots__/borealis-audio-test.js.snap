exports[`BorealisAudio Tests renders an audio 1`] = `
<audio
  className="audio-player"
  controls={true}>
  <source
    src={undefined}
    type="audio/mpeg" />
  Your browser does not support the audio element.
</audio>
`;
